cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,45 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
-  before(() => {
+const userName = 'MarkelMarch14';
```

Comment:
this is the registration flow, in case when you don't clean DB you cannot register a new user with the same creds. So use the random user generation function